backend/api-service/src/backend_firestore.go

```diff
@@ -24,6 +24,12 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const alertsStr = "alerts"
```

Comment:
nit:  it would be useful to name these after what they represent (Firestore collections), so perhaps `alertsCollection` would be a more appropriate name.

Reply:
thanks - renamed as per the suggestion.